tests/testthat/test-api-status.R

```diff
@@ -1,3 +1,35 @@
 test_that("gh_api_status() works", {
+  if (!nzchar(Sys.getenv("REAL_REQUESTS"))) {
+   app <- presser::new_app()
+      app$get("/", function(req, res) {
+        res$send_json(
+          list( components =
```

Comment:
in this case I wrote the object to be returned by hand. This way my PR shows two methods (in one test the response is written in the test file, in the other case there's a JSON file that's loaded)